cribl_stream/manifest.json

```diff
@@ -0,0 +1,59 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "2ef15aea-af91-4769-940c-2b124e4d04a6",
+  "app_id": "cribl-stream",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect observability data in a vendor-neutral data telemetry pipeline",
+    "title": "Cribl Stream",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Cribl Stream Logo",
+        "image_url": "images/CriblStream.FULL_COLOR.png"
+      },
+      {
+        "media_type": "video",
+        "caption": "Observability Changes Everything",
+        "image_url": "images/observability_changes.png",
+        "vimeo_id": 567294419
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Category::Marketplace",
```

Comment:
```suggestion
```
This will live on the Integrations page, so Marketplace is not relevant here. Please add any relevant categories from this list: https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags